package.json

```diff
@@ -92,6 +89,9 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@uniswap/sdk": "^2.0.6",
```

Comment:
Do we need all these to be dependencies? I think we are only using `v2-core` in the library.